stable/ark/values.yaml

```diff
@@ -1,6 +1,6 @@
 image:
   repository: gcr.io/heptio-images/ark
-  tag: v0.8.2
+  tag: v0.9.1
```

Comment:
Please update references in the README to be consistent with the values.yaml

Reply:
ah, should be fixed now. I saw another error in the readme while looking at this, which i also fixed in my commit.